[2020-09-10T09:17:30.794] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T09:21:35.578] [ERROR] app - 登录已过期
[2020-09-10T09:22:28.113] [ERROR] app - 登录已过期
[2020-09-10T09:22:28.177] [ERROR] app - 登录已过期
[2020-09-10T09:22:33.352] [ERROR] app - 登录已过期
[2020-09-10T09:22:33.393] [ERROR] app - 登录已过期
[2020-09-10T09:22:37.511] [ERROR] app - 密码错误
[2020-09-10T09:29:46.479] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T09:30:40.708] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T10:02:32.152] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T10:10:01.054] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T10:10:02.081] [ERROR] app - Error: getaddrinfo ENOTFOUND cdb-gtz432y7.gz.tencentcdb.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:36:13)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14 {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'cdb-gtz432y7.gz.tencentcdb.com',
  fatal: true
}
[2020-09-10T10:13:31.455] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:40:51)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\response.js:12:9)
    at async cors (D:\hstigger\hstigger-server\node_modules\koa2-cors\dist\index.js:91:9)
    at async D:\hstigger\hstigger-server\node_modules\koa-log4\koa-logger.js:86:7 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T10:13:32.482] [ERROR] app - Error: getaddrinfo ENOTFOUND cdb-gtz432y7.gz.tencentcdb.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:36:13)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14 {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'cdb-gtz432y7.gz.tencentcdb.com',
  fatal: true
}
[2020-09-10T10:30:36.481] [ERROR] app - 用户未登录
[2020-09-10T10:30:36.517] [ERROR] app - 用户未登录
[2020-09-10T10:31:32.855] [ERROR] app - 用户未登录
[2020-09-10T10:31:32.893] [ERROR] app - 用户未登录
[2020-09-10T10:37:34.737] [ERROR] app - Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:409:13)
    at Object.onceWrapper (events.js:420:28)
    at Socket.emit (events.js:314:20)
    at Socket._onTimeout (net.js:483:8)
    at listOnTimeout (internal/timers.js:551:17)
    at processTimers (internal/timers.js:494:7)
    at Protocol._enqueue (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\hstigger\hstigger-server\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at Connection.connect (D:\hstigger\hstigger-server\node_modules\mysql\lib\Connection.js:116:18)
    at D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:69:18
    at new Promise (<anonymous>)
    at Client_MySQL.acquireRawConnection (D:\hstigger\hstigger-server\node_modules\knex\lib\dialects\mysql\index.js:64:12)
    at create (D:\hstigger\hstigger-server\node_modules\knex\lib\client.js:291:39)
From previous event:
    at processImmediate (internal/timers.js:458:21)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:204:8)
From previous event:
    at Sync.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\sync.js:66:17)
From previous event:
    at Child.<anonymous> (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:713:12)
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
From previous event:
    at Child.fetch (D:\hstigger\hstigger-server\node_modules\bookshelf\lib\model.js:702:19)
    at LoginWithPassword (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:92:10)
    at Object.out.Post (D:\hstigger\hstigger-server\hstigger-server\restful\v0\sessions.js:122:21)
    at RestfulAPIMethods.<computed> (D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:32:60)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\hstigger-server\routes\restful.js:97:19
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\hstigger\hstigger-server\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at module.exports (D:\hstigger\hstigger-server\hstigger-server\middlewares\auth.js:36:13)
    at dispatch (D:\hstigger\hstigger-server\node_modules\koa-compose\index.js:42:32)
    at D:\hstigger\hstigger-server\node_modules\koa-bodyparser\index.js:74:14 {
  errorno: 'ETIMEDOUT',
  code: 'ETIMEDOUT',
  syscall: 'connect',
  fatal: true
}
[2020-09-10T10:38:14.607] [ERROR] app - 用户未登录
[2020-09-10T10:38:14.651] [ERROR] app - 用户未登录
[2020-09-10T10:39:33.848] [ERROR] app - 用户未登录
[2020-09-10T10:39:33.886] [ERROR] app - 用户未登录
[2020-09-10T10:43:08.413] [ERROR] app - 用户未登录
[2020-09-10T10:43:08.461] [ERROR] app - 用户未登录
[2020-09-10T10:43:34.730] [ERROR] app - 用户未登录
[2020-09-10T10:43:34.789] [ERROR] app - 用户未登录
[2020-09-10T10:44:42.288] [ERROR] app - 用户未登录
[2020-09-10T10:44:42.324] [ERROR] app - 用户未登录
[2020-09-10T10:45:07.891] [ERROR] app - 用户未登录
[2020-09-10T10:45:07.930] [ERROR] app - 用户未登录
[2020-09-10T13:54:40.355] [ERROR] app - 用户名不存在
[2020-09-10T13:54:49.451] [ERROR] app - 用户名不存在
[2020-09-10T14:17:59.210] [ERROR] app - 用户名不存在
